src/ducks/apps/components/PermissionsList.jsx

```diff
@@ -48,7 +48,14 @@ const getProcessedPermissions = (t, app) => {
     const linxoType = 'io.cozy.accounts'
     external.push(
       <Permission
-        description={t('permissions.linxo')}
+        description={
+          app.partnership && app.partnership.name && app.partnership.domain
+            ? t('permissions.banking', {
+                name: app.partnersip.name,
+                domain: app.partnership.domain
+              })
+            : t('permissions.linxo')
```

Comment:
Is it good to have a locale key named `linxo` ?

Reply:
Since it's already in the text here I think we can keep it